cmd/batch.go

```diff
@@ -356,7 +393,7 @@ func (b *standardBatchPipeline) WriteFile(
 		return nil
 	}
 
-	return b.WriteTemplate(writer, string(fileBytes), true, templateData)
+	return b.WriteTemplate(writer, string(fileBytes), 1, 0, templateData)
```

Comment:
And I am not sure what this part of the code writes to the file, so I know what kind of newlines config to put.